packages/chord-mark/src/renderer/components/tpl/song.js

```diff
@@ -1,4 +1,10 @@
 const render = ({ song }) => {
-	return `<div class="cmSong">${song}</div>`;
+	/**
+	 * The extra opening and closing div accounts for the simple section 
+	 * wrapping logic, which wraps sections in <div class="cmSection ..."> for
+	 * the purpose of styling with themes. Yes, this does spawn one needless div 
+	 * at the start of the song.
+	 */
+	return `<div class="cmSong"><div>${song}</div></div>`;
```

Comment:
I'm not a big fan of this. I would prefer some extra logic in the rendering loop to properly handle opening and closure of section wrappers.

Reply:
It is super hacky for sure. Ok.